src/Sylius/Bundle/ShopBundle/templates/account/reset_password.html.twig

```diff
@@ -1,7 +1,5 @@
 {% extends '@SyliusShop/shared/layout/base.html.twig' %}
 
-{% form_theme form '@SyliusShop/Form/theme.html.twig' %}
-
 {% block title %}{{ 'sylius.ui.reset_password'|trans }} | {{ parent() }}{% endblock %}
 
 {% block content %}
```

Comment:
I would be in favour of putting all the content in Twig Hook `sylius_shop.account.reset_password`